frontend/src/app/testQueue/TestQueue.tsx

```diff
@@ -218,26 +214,15 @@ const TestQueue: React.FC<Props> = ({ activeFacilityId }) => {
             onExiting={onExiting}
             timeout={transitionDuration}
           >
-            {testCardRefactorEnabled ? (
-              <TestCard
-                testOrder={testOrder}
-                devicesMap={devicesMap}
-                facility={facility}
-                refetchQueue={refetch}
-                removePatientFromQueue={removePatientFromQueue}
-                startTestPatientId={startTestPatientId}
-                setStartTestPatientId={setStartTestPatientId}
-              />
-            ) : (
-              <QueueItem
```

Comment:
⚠️ removed any `QueueItem` component related files as this was the only place we were using it